### PR TITLE
Updated Collective data returned for open source and meetups on /homepage

### PR DESCRIPTION
**Pull request by carlosascari**
Includes backers list of groups
Includes yearlyIncome for each group

This is to complete the home page, the data here will fill the CollectiveCard component's footer.

### CODE REVIEW

File: server/controllers/homepage.js [hunk 1]
```diff
@@ -45,8 +84,8 @@ module.exports = function(app) {
       getTotalCollectives(),
       getTotalDonors(),
       queries.getTotalDonations(),
-      getGroupsByTag('open source'),
-      getGroupsByTag('meetup')
+      getGroupsByTagForCollectiveCard('open source'),
+      getGroupsByTagForCollectiveCard('meetup')
```
Comment (xdamman) [inline server/controllers/homepage.js]: Why did you rename it? Just change the current implementation of `getGroupsByTag`. Otherwise we will have dead code.

Comment (carlosascari) [inline server/controllers/homepage.js]: I did not rename the function, I created another function that extends `getGroupsByTag`, specifically to supply the information that is displayed at the footer of the CollectiveCard component.

https://github.com/OpenCollective/opencollective-api/pull/478/files#diff-9c36a551a062a029a91b82d5299327ebR25
